process_under_control.py

```diff
@@ -0,0 +1,27 @@
+def check_process_control(m, data, UCL, CL, LCL, UCL_x, C_x, LCL_x):

```

Comment:
Da die Grenzen nicht innerhalb der Funktion berechnet werden, wird der Parameter `m` nicht benötigt.
```suggestion
def check_process_control(data, UCL, CL, LCL, UCL_x, C_x, LCL_x):

```